.travis.yml

```diff
@@ -2,121 +2,264 @@
 # project configuration #
 #########################
 
-# C++ project
-language: cpp
+language: python
 
-dist: trusty
 sudo: required
 
 ################
 # build matrix #
 ################
 
+#        brew install cmake
+#        brew install openmpi
+#        brew install --without-python boost
+#        brew install lapack
+#        brew upgrade cmake
+#        brew upgrade openmpi
+#        cmake --version
+
 matrix:
   include:
+    - os: osx
+      osx_image: xcode9.4
+      language: generic
+      env: PYTHON=2.7 CXX=clang++
 
-  # clang sanitizer
-  - os: osx
-    osx_image: xcode9.4
-    compiler: clang
-    env:
-      - CMAKE_OPTIONS=-DNETKET_Sanitizer=ON
-
-  - os: osx
-    osx_image: xcode8.3
-    install:
-      - brew upgrade python
-
-  - os: osx
-    osx_image: xcode9.4
-
-  - os: linux
-    before_install:
-      - pyenv global $(pyenv whence 2to3)  # activate all python versions
-      - PY_CMD=python3
-      - $PY_CMD -m pip install --user --upgrade pip wheel
-    compiler: gcc
-    env:
-      - PYTHON=3.6
-      - COMPILER=g++-5
-      - CMAKE_OPTIONS=-DUSE_LAPACK=ON
-    addons:
-      apt:
-        sources: ['ubuntu-toolchain-r-test']
-        packages: ['g++-5','mpich','libmpich-dev', 'libboost-all-dev',
-                   'liblapack3','libblas-dev', 'liblapack-dev','liblapacke-dev','python-numpy', 'python-scipy', 'python-matplotlib',
-                   'python-tk']
-
-  - os: linux
-    before_install:
-      - pyenv global $(pyenv whence 2to3)  # activate all python versions
-      - PY_CMD=python3
-      - $PY_CMD -m pip install --user --upgrade pip wheel
-    compiler: gcc
-    env:
-      - PYTHON=3.6
-      - COMPILER=g++-6
-      - CMAKE_OPTIONS=-DUSE_LAPACK=ON
-    addons:
-      apt:
-        sources: ['ubuntu-toolchain-r-test']
-        packages: ['g++-6','mpich','libmpich-dev', 'libboost-all-dev','liblapack3',
-                   'libblas-dev', 'liblapack-dev','liblapacke-dev','python-numpy', 'python-scipy', 'python-matplotlib',
-                   'python-tk']
-
-  - os: linux
-    before_install:
-      - pyenv global $(pyenv whence 2to3)  # activate all python versions
-      - PY_CMD=python3
-      - $PY_CMD -m pip install --user --upgrade pip wheel
-    compiler: clang
-    env:
-      - PYTHON=3.6
-      - COMPILER=clang++-5.0
-      - CXXFLAGS=-std=c++1z
-      - CMAKE_OPTIONS=-DUSE_LAPACK=ON
-    addons:
-      apt:
-        sources: ['ubuntu-toolchain-r-test', 'llvm-toolchain-trusty-5.0']
-        packages: ['g++-6', 'clang-5.0', 'mpich','libmpich-dev', 'libboost-all-dev',
-                   'liblapack3','libblas-dev', 'liblapack-dev','liblapacke-dev','python-numpy', 'python-scipy', 'python-matplotlib',
-                   'python-tk']
-
-
-cache:
-  apt: true
-  directories:
-    - $HOME/Library/Caches/Homebrew
+    - os: osx
+      osx_image: xcode10.1
+      language: generic
+      env: PYTHON=3.6.0 CXX=clang++
```

Comment:
At least one of the runs should happen with the sanitizer on, can you put it back here?

Reply:
I'd prefer to have OS X build actually working first... But then sure, I'm open to adding however many flags you'd like.